lunes_cms/static/js/toggle_document_form_fields.js

```diff
@@ -22,76 +22,79 @@ $(document).ready(() => {
   $("#id_word_type").trigger("change");
 });
 
-function removeDuplicationCheckMessage(){
-  var existingMessage = document.getElementById("result");
-
-  if (existingMessage) {
-    existingMessage.remove();
-  }
-}
+// Detect changes in the "word" field and search for a duplicate
```

Comment:
I hate to say it, but I think that the refactoring broke the notification about duplicated words for me...
When I create a new word that already exists in the system, I don't get a warning anymore. Is this wanted?

Reply:
What.........?! and it's of course not wanted......... 😅 🙈 A fox must have used an evil magic or something 😅 🙈

It should be fixed now 😅 🙈

Reply:
Verified that the alarm pops up again, so I'll close this conversation.